postgres/src/test/resources/shop_schema.sql

```diff
@@ -189,4 +195,12 @@ values
     ('852E2DC9-4EC3-4225-A6F7-4F42F8FF728E', 'D5137D3A-894A-4109-9986-E982541B434F', 1, 45.45),
     ('D6D8DDDC-4B0B-4D74-8EDC-A54E9B7F35F7', 'D5137D3A-894A-4109-9986-E982541B434F', 2, 50.00),
     ('2C3FC180-D0DF-4D7B-A271-E6CCD2440393', 'D5137D3A-894A-4109-9986-E982541B434F', 2, 50.00),
-    ('5883CB62-D792-4EE3-ACBC-FE85B6BAA998', 'D5137D3A-894A-4109-9986-E982541B434F', 1, 55.00);
\ No newline at end of file
+    ('5883CB62-D792-4EE3-ACBC-FE85B6BAA998', 'D5137D3A-894A-4109-9986-E982541B434F', 1, 55.00);
+
+insert into timestamp_test
```

Comment:
I actually had in mind inserting rows using a zio-sql query, but doesn't look like we support that at the moment. Can you please add the created_timestamp and created_timestamp_string columns to the customer table instead of creating this new one (which looks like the approach you were taking before) and add a note to remove the created_timestamp_string column once #320 is done?

Reply:
> I actually had in mind inserting rows using a zio-sql query, 
Yes I can do that. my original plan was to do that as well. I'll make a note and your requested change!

- [ ] using existing customers table
- [ ] add note